Desktop version

pulgosodog opened the pull request:
Changes made according to the **Portfolio: desktop version** activity:

1. Add media queries to the intro section
2. Add media queries to the works section
3. Add media queries to the about myself section
4. Add media queries to the contact form section

pulgosodog committed:
Fix section overlay with bigger screen width

Also set responsive values to **p**s on the document

pulgosodog committed:
Add media queries to intro section

pulgosodog committed:
Add media queries styles to works section

pulgosodog committed:
Finish style with media queries specifics on cards

pulgosodog committed:
Finish media queries on about myself section

pulgosodog committed:
Add media queries to contact form section

pulgosodog committed:
Fix mark linter errors

pulgosodog committed:
Fix mark linter errors

pulgosodog committed:
Fix mark linter errors

taldr27 commented on index.html:
```diff
@@ -16,20 +16,30 @@
-            work
-            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact
-            me.
+        <h1 class="margin h1-query intro-margin">I'm Gabriel. <br>Glad to see you!</h1>
+        <p class="margin intro-margin description">I’m a software developer! I can help you build a product , feature or
+            website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hesitate to contact me.
```
- [ ] Great job on your project! I noticed an issue **with the alignment when switching from desktop to mobile view**. The content is pushed **to the left and there's an empty space on the right side in desktop view**. I took a look at your code and it seems like the issue is caused by the `width` of one of your hidden images set to `93%` `(style.css - line 153)`. To fix this, you can remove the `width` or adjust it to a smaller value. Additionally, try to avoid using `vw` units for sizes and instead use more fixed units like **pixels or percentages**. This **will help maintain the consistency of your layout across different screen sizes**. Keep up the good work! 🎯

  _Watch the next video for a better understanding_ 🎬
  
  [scrnli_5_7_2023_12-47-55 AM.webm](https://user-images.githubusercontent.com/107952724/236660194-4f56b7c3-500e-43cb-a321-a87ef5cb1f69.webm)

pulgosodog replied:
Hey! I really appreciate the review, I was not sure how to address it.
I also set the width and height of desktop images to 0 while on mobile ratios.

pulgosodog replied:
Issue addressed!

taldr27 reviewed (changes requested):
# Status: Required Changes ♻️♻️
Hello @pulgosodog, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

![almost-there](https://media.giphy.com/media/3ohjVa9eUyIaOD5Fsc/giphy.gif)
## Highlights 🎉
✅ Professional and customized README. 🙋‍♂️
✅ Linters tests passing. 💯
✅ Correct use of GitHub Flow.
✅ Good job with your design.
✅ Nice use of background images for the desktop version. 🎯

## Required Changes ♻️
_Check the comments under the review._


------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_

pulgosodog committed:
Fix css style extra space on mobile and desktop

Set width and height of desktop images to 0 while on mobile ratios.
Erase card-img width to fix the extra space on the desktop version

pulgosodog committed:
Fix linter mark error

pulgosodog committed:
Add width and height to the mobile images

pulgosodog committed:
Fix linter mark issues

Strangeal reviewed (approved):
Hi @pulgosodog, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/JoSiL3L4ON8DlyVxv0/giphy.gif)

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
